### PR TITLE
Password must be strong with specific rules issue#337

### PR DESCRIPTION
**Pull request by kapilG0**
## Description
Now Register button is disabled if password is not good
## Related Issue
#337 

## Screenshots / GIFs (if applicable)
[Attach any relevant screenshots or GIFs demonstrating the changes]

## Checklist:
- [x] I have performed a self-review of my code
- [ ] I have added/updated relevant documentation (if needed)
- [x] I have tested the changes locally and they function as expected
- [ ] I have ensured my code follows the project's coding standards

## Additional Notes
[Add any additional notes or context about the changes made]

### CODE REVIEW

**Comment by codewarnab** [comment]
please mention related issue

**Comment by kapilG0** [comment]
> please mention related issue

Done

**Comment by rishicds** [comment]
Hi @kapilG0 , please update your branch

**Comment by kapilG0** [comment]
> Hi @kapilG0 , please update your branch

Do you mean I should sync my forked repo then raise pr ?

File: src/pages/user/UserRegistration.jsx [hunk 1]
```diff
@@ -133,13 +148,15 @@ function LoginRegisterForm() {
...
                     messageDuration={3000}
+                    disabled={errorMessage==='Is Not Strong Password , Should contain atleast one lowecase letter and atleast one uppercase letter and atleast one number and atleast one symbol and min length should be 8'}
                     onClick={async () =>
                       registration(
                         name,
                         phone_number,
                         email,
                         password,
                         setError,
+                        setErrorMessage,
```
Comment (codewarnab) [inline src/pages/user/UserRegistration.jsx:159]: why did you passed this in this registartion function ?

**Review by codewarnab — CHANGES_REQUESTED** [review]
@kapilG0 sync your branch resolve conflicts then commit again

**Comment by kapilG0** [comment]
> @kapilG0 sync your branch resolve conflicts then commit again

Hello @codewarnab I raised new pull request https://github.com/Innovateninjas/Paws-frontend/pull/351. Please check after sync my forked repo. Thanks

**Pull request closed without merging**